data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -672,15 +671,227 @@
                     [/have_unit]
                 [/or]
             [/filter_condition]
+            [if]
+                # Only flying units can reach this location and the only flying units the player should have access to besides Krawg is the Gryphon Rider line
+                [variable]
+                    name=unit.id
+                    equals=Krawg
+                [/variable]
+                [then]
+                    [message]
+                        speaker=Krawg
+                        message= _ "Wha’?"
+                    [/message]
+                    [message]
+                        speaker=narrator
+                        message= _ "Krawg puzzled over a stone tile sitting loose on the small mesa. Beneath a spattering of bat guano, there was a glowing rune. Krawg didn’t know what to make of it, so the gryphon left it for now."
+                        image=wesnoth-icon.png
+                    [/message]
+                [/then]
+                [else]
+                    [message]
+                        speaker=$unit.id
+                        message= _ "The gryphon rider puzzled over a stone tile sitting loose on the small mesa. Beneath a spattering of bat guano, there was a glowing rune. The rider didn’t know what to make of it, so the rider left it for now."
```

Comment:
The grammar feels a bit odd, as if "rider" refers to the gryphon.

"Neither mount nor rider knew what to make of it, so they left it for now."